lumigator/python/mzai/schemas/README.md

```diff
@@ -0,0 +1 @@
+# TODO
```

Comment:
does this also need to be moved and renamed?

Reply:
We'll keep this readme with some doc on the schemas.